js/tests/friend-support_test.mjs

```diff
@@ -0,0 +1,125 @@
+import { once } from 'node:events'
+import * as cp from 'node:child_process'
```

Comment:
Since only spawn is being used, wouldn't it be better to just destructure it `import { spawn } from ...`?